execution/src/simulate.rs

```diff
@@ -115,7 +115,13 @@ pub fn call_transaction(
                 verify_input.traversal_context,
                 &code_storage,
             )?;
-            Ok(bcs::to_bytes(&outcome.return_values)?)
+            Ok(bcs::to_bytes(
+                &outcome
+                    .return_values
+                    .into_iter()
+                    .map(|(bytes, _ty)| bytes)
```

Comment:
Feels it could be useful to state the rationale here, just like in the PR description.

Reply:
Good idea, added in [3ad3a76](https://github.com/MovedNetwork/op-move/pull/342/commits/3ad3a76e161100b4996b3dc323fbd00b25563b48)